File: cash.py
from cs50 import get_float

amount = int(get_float("Change: ")*100)
count = 0
while(True):
    if(amount > 0):
        break
    else:
        amount = int(get_float("Change: ")*100)
while(amount > 0):
    if(amount >= 25):
        amount -= 25
        count += 1
    elif(amount < 25 and amount >= 10):
        amount -= 10
        count += 1
    elif(amount < 10 and amount >= 5):
        amount -= 5
        count += 1
    elif(amount < 5 and amount >= 1):
        amount -= 1
        count += 1
print(f"{count}")

File: cash.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int a = 0;
    int b = 0;
    int c = 0;
    int d = 0;
    int cash;
    do
    {
        cash = get_int("Change Owed: ");
    }
    while(cash < 0);
    while(cash>=25)
    {
        cash = cash - 25;
        a = a + 1;
        printf("a is: %i\n", a);
    }
    while(cash>=10)
    {
        cash = cash - 10;
        b = b + 1;
        printf("b is: %i\n", b);
    }
    while(cash>=5)
    {
        cash = cash - 5;
        c = c + 1;
        printf("c is: %i\n", c);
    }
    while(cash>=1)
    {
        cash = cash - 1;
        d = d + 1;
        printf("d is: %i\n", d);
    }
    int e = a+b+c+d;
    printf("%i\n", e);
    return(0);
}
